ddl/partition.go

```diff
@@ -2154,6 +2159,363 @@ func (w *worker) onExchangeTablePartition(d *ddlCtx, t *meta.Meta, job *model.Jo
 	return ver, nil
 }
 
+func checkReorgPartition(t *meta.Meta, job *model.Job) (*model.TableInfo, []model.CIStr, *model.PartitionInfo, []model.PartitionDefinition, []model.PartitionDefinition, error) {
```

Comment:
Will this function be used in the next a few PRs?

Reply:
Yes, but if it is not referenced in this PR, let me remove it so it is introduced only when it is used.

Reply:
Removed.